docs/reference/webhook-reference.md

```diff
@@ -0,0 +1,92 @@
+# Webhook
+
+The Webhook class enables you to view, test, and modify custom event notifications with [webhooks](webhooks.md).
+
+```python
+webhook = up42.initialize_webhook(webhook_id="1df1ebb0-78a4-55d9-b806-15d22e391bd3")
```

Comment:
is this a real id?

Reply:
hmmm it shouldn't be. I double checked with my own webhooks, and none of them have that ID. If it seems like it could be a real one, I can replace with one of the webhook IDs the [API reference](https://docs.up42.com/developers/api#tag/Webhooks/operation/updateWebhook) uses: `d290f1ee-6c54-4b01-90e6-d701748f0851`. @seedlit

Reply:
Not a big deal, but yeah, please use fake ids.